app/assets/stylesheets/top-bar.scss

```diff
@@ -48,6 +48,18 @@
 .top-bar--logo {
   display: flex;
 
+  &--logged-out {
+    width: auto;
+
+    @media (min-width: $breakpoint-m) {
+      width: 2fr;
+    }
+
+    @media (min-width: $breakpoint-l) {
+      width: 240px;
```

Comment:
This adds the space between logo and the search BUT it makes the space clickable.

Reply:
Noted!